data/bouncmpe/people/ozgovde-atay.json

```diff
@@ -7,7 +7,8 @@
     "former-postdoc",
     "former-cmpe140",
     "cmpe250",
-    "advisor-1"
+    "advisor-1",
+    "former-cmpe350"
```

Comment:
Do we know Atay hoca taught CMPE350 in the past?

Reply:
![Ekran görüntüsü 2024-10-13 224408](https://github.com/user-attachments/assets/0a762dc6-030c-4b53-8a5b-b858d5472410)
I saw that in website. Also, there is no info before 2010